desdeo_problem/testproblems/CarCrash.py

```diff
@@ -0,0 +1,91 @@
+from desdeo_problem.problem.Variable import Variable
+from desdeo_problem.problem.Objective import ScalarObjective
+from desdeo_problem.problem.Problem import MOProblem, ProblemBase
+
+import numpy as np
+
+def car_crash_problem(var_iv: np.array = ([2, 2, 2, 2, 2])) -> MOProblem:
+    """The crash safety design problem with 3 objectives.
+
+    Liao, X., Li, Q., Yang, X., Zhang, W. & Li, W. (2007).
+    Multiobjective optimization for crash safety design of vehicles
+    using stepwise regression model. Structural and multidisciplinary
+    optimization, 35(6), 561-569. https://doi.org/10.1007/s00158-007-0163-x
+
+    Arguments:
+        var_iv (np.array): Optional, initial variable values. Must be between
+            1 and 3. Defaults are [2, 2, 2, 2, 2].
+
+    Returns:
+        MOProblem: a problem object.
+    """
+
+    for x in var_iv:
+        if 3 < x or x < 1:
```

Comment:
You can compare all the values in a numpy array at once, like

```
if any(3 < var_iv) or any(var_iv < 1):
    # do stuff
```

Usually loops should be avoided with numpy arrays and rely on casting instead.